.prettierignore

```diff
@@ -0,0 +1,4 @@
+build
+dist
+node_modules
+assets
```

Comment:
GitHub needs empty line at the end of each file you want to add

Reply:
Added empty line for each file (also changed settings for my IDE)